client-app/src/pages/account/views/OrdersList.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div>
+    <h2>OrdersList</h2>
+    <b-table striped
+             hover
+             :items="orders"
+             :fields="fields"></b-table>
+  </div>
+</template>
+
+<script>
+import { GET_ORDERS } from "../store/modules/orders/action-types";
```

Comment:
Not important, but it's always better to use ts aliases, like `@account/store/modules/orders/action-types`. I'll look at eslint, may be it may fix it automatically

Reply:
I agree